Dockerfile

```diff
@@ -1,3 +1,28 @@
 FROM jenkinsci/jnlp-slave
 
-## @TODO: Adde required dependencies
\ No newline at end of file
+USER root
+
+## ===@TODO: Add required dependencies==
+
+ARG HELM_VERSION=v2.1.3
+
+RUN mkdir /home/helm && chmod -R 777 /home/helm
+
+ENV HELM_HOME=/home/helm
+
+RUN curl https://kubernetes-helm.storage.googleapis.com/helm-$HELM_VERSION-linux-386.tar.gz | tar xz -C /tmp \
+    && chmod 755 /tmp/linux-386/helm \
+    && mv /tmp/linux-386/helm /usr/bin
+
+
+## ====================================
+
+USER jenkins
+
+ADD helm.repo /tmp/helm.repo
+
+RUN helm init -c  \
+    && cat /tmp/helm.repo | xargs -n 2 helm repo add $1 $2 \
```

Comment:
the `$1 $2` will not work like that; just drop them. it might work this way, but that's b/c `$1` and `$2` are null.
```
e@Cirrus:~/Dropbox/Dev/sagan$ cat /tmp/test.txt | xargs -n 2 echo "[$1]" "[$2]"
[] [] 1 2
[] [] 3 4
[] [] 5 6
```